lib/hanami/generators/app/application.rb.tt

```diff
@@ -200,7 +200,28 @@ module <%= config[:classified_app_name] %>
       #  * http://content-security-policy.com/
       #  * https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Using_Content_Security_Policy
       #
-      security.content_security_policy "default-src 'none'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self'; font-src 'self';"
+      # Content Security Policy references:
+      #
+      #  * https://developer.mozilla.org/en-US/docs/Web/Security/CSP/CSP_policy_directives
+      #
+      security.content_security_policy %{
+        form-action 'self';
+        referrer origin-when-cross-origin;
+        reflected-xss block;
+        manifest-src 'self';
```

Comment:
@joneslee85 According to [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/Security/CSP/CSP_policy_directives#manifest-src):

> The `manifest-src` directive specifies which manifest can be applied to the resource.
> **Note**: If this directive is absent, the user agent will look for the `default-src` directive.

This should be a ["source list"](https://developer.mozilla.org/en-US/docs/Web/Security/CSP/CSP_policy_directives) (aka URL), not a keyword like `'self'`.

Because we don't support [Web App Manifest](https://www.w3.org/TR/appmanifest/), we should remove this entry at all, and let the browser to fallback to `default-src`, as described by Mozilla docs above.